### ISSUE TITLE
[CE 2.0.7]-Can't adjust inventory for bundled products

### ISSUE THREAD

**Issue by Ctucker9233** [magento/magento2#5092]
The Quantity field for inventory is greyed out with a 0 in the field. Even though it is set to "In Stock" the product does not display on frontend.

**Comment by Ctucker9233** [comment]
Even enabling display out of stock items doesn't work. It's like they don't exist even though I can clearly see the products in the backend.

**Comment by Ctucker9233** [comment]
Any one have any suggestions to fix this problem? I can upload screenshots if it helps.

**Comment by andimov** [comment]
To allow us research the issue, please, provide description according to the template.

**Comment by Ctucker9233** [comment]
I apologize for the delay in answering. This will be closed as I have solved the problem although I would like to suggest that the document be updated to ore clearly reflect that the "add options" portion is for adding the simple products that are part of the bundle. Only after "options" are added will the bundled product show up on the frontend I had to search quite a few places before I found that this was why my bundled product wasn't appearing.
